### PR TITLE
add nfts_by_issuer and fix links

### PR DESCRIPTION
**Pull request by DennisDawson**
Add nfts_by_issuer reference and add links for nft_history.

### CODE REVIEW

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+<!-- To DO: Add an example command to the assets/js/apitool-methods-ws.js file. The WebSocket Tool requires access to a publicly available Clio server.
+[Try it! >](websocket-api-tool.html#nft_history)-->
+
+The request contains the following parameters:
+
+| `Field`        | Type    | Description                    |
+|:---------------|:--------|:-------------------------------|
+| `issuer`       | String  | A unique identifier for the account, most commonly the account's address.  |
+| `marker`       | Marker  | Value from a previous paginated response. Resume retrieving data where that response left off. This value is NOT stable if there is a change in the server's range of available ledgers; meaning if you are querying the “validated” ledger it is possible that new NFTs are created during your paging. |
```
Comment (shawnxie999) [inline docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md]: marker should be optional too

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+```json
+{"result":{
+  "nft_id":"00082710D16560292CA49840AE2DDD7A2CAA50C386BC1B6704F1BB1A004D027F","ledger_index":5046924,"owner":"rLaBCoaMQqXzHDmiGs2Qv2JA2bg3Yvzxyt",
+  "is_burned":false,
+  "flags":8,
+  "transfer_fee":10000,
+  "issuer":"rLaBCoaMQqXzHDmiGs2Qv2JA2bg3Yvzxyt",
+  "nft_taxon":0,"nft_serial":5046911,"validated":true,"uri":"697066733A2F2F62616679626569676479727A74357366703775646D37687537367568377932366E6634646675796C71616266336F636C67747179353566627A6469"},"id":1,"status":"success","type":"response","warnings":[{"id":2001,"message":"This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request"}]
+}
```
Comment (shawnxie999) [inline docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md:75]: I don't think this JSON is correct. This looks like a nft_info response. Here is a correct example

```
{
   "result": {
      "issuer": "rfXeQv31yWMrhhPxMHZRzQqhw5mQrcuici",
      "limit": 50,
      "ledger_index": 1534,
      "nfts": [
         {
            "nft_id": "00080000479C76BC5174816A938ABF667E67D851140BFE03F068FA97000005FB",
            "ledger_index": 1533,
            "owner": "rfXeQv31yWMrhhPxMHZRzQqhw5mQrcuici",
            "is_burned": false,
            "uri": "",
            "flags": 8,
            "transfer_fee": 0,
            "issuer": "rfXeQv31yWMrhhPxMHZRzQqhw5mQrcuici",
            "nft_taxon": 1,
            "nft_serial": 1531
         }
      ],
      "validated": true,
      "status": "success"
   },
   "warnings": [
      {
         "id": 2001,
         "message": "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request"
      }
   ]
}
```

(I'm happy to provide with such JSON example in the future, I could generate them locally and give them to you, if that's easier)

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+The response follows the [standard format][], with a successful result containing the following fields:
+
+| `Field`            | Type                       | Description                |
+|:-------------------|:---------------------------|:---------------------------|
+| `issuer`           | String                     | Issuer's account ID.       |
+| `nfts`             | Array&lt;Object&gt;              | A list of NFTs issued by the account. The order of the NFTs is not associated with the date the NFTs were minted. |
+| `marker`           | Marker                     | _(Optional)_ Server-defined value indicating the response is paginated. Pass this to the next call to resume where this call left off. If this field is not returned, you know that you no longer need to make calls to this API. |
+| `limit`            | Integer                    | The `limit`, as specified in the request. |
+| `nft_taxon`        | Integer                    | The `nft_taxon` as specified in the request. |
```
Comment (shawnxie999) [inline docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md]: taxon is optional as well.

**Review by shawnxie999 — COMMENTED** [review]
Just a few little things...otherwise looks good

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  "nft_taxon":0,"nft_serial":5046911,"validated":true,"uri":"697066733A2F2F62616679626569676479727A74357366703775646D37687537367568377932366E6634646675796C71616266336F636C67747179353566627A6469"},"id":1,"status":"success","type":"response","warnings":[{"id":2001,"message":"This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request"}]
+}
+```
+
+The response follows the [standard format][], with a successful result containing the following fields:
+
+| `Field`            | Type                       | Description                |
+|:-------------------|:---------------------------|:---------------------------|
+| `issuer`           | String                     | Issuer's account ID.       |
+| `nfts`             | Array&lt;Object&gt;              | A list of NFTs issued by the account. The order of the NFTs is not associated with the date the NFTs were minted. |
```
Comment (shawnxie999) [inline docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md:83]: We should add a note saying that the format of each NFT in the `nfts` array is exactly the same as the response of a `nft_info`

image

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md (hunk 2 above)
Comment (DennisDawson) [reply docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md:75]: Yes, thank you! it wouldn't hurt to include examples in the spec, too.

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md (hunk 4 above)
Comment (DennisDawson) [reply docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md:83]: Added a link to `nft_info`.

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md (hunk 1 above)
Comment (DennisDawson) [reply docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md]: Fixed.

File: docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md (hunk 3 above)
Comment (DennisDawson) [reply docs/references/http-websocket-apis/public-api-methods/clio-methods/nfts_by_issuer.md]: Fixed.

**Review by shawnxie999 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**